sdk/samples/iot/CMakeLists.txt

```diff
@@ -161,6 +161,7 @@ if (MSVC)
     set_target_properties(
         paho_iot_hub_c2d_sample
         paho_iot_hub_methods_sample
+        paho_iot_hub_pnp_component_example
```

Comment:
fyi, @danewalton

Reply:
ah yes would you look at that i forgot something haha. my bad. thanks for the heads up!

Reply:
Once this PR is merged, it will be easier to stay green (at least for this particular instance), because CI would fail due to the MSVC warning.